src/ignoredTextFiles.ts

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
Thoughts on changing this file to something like `knownIgnores` or `excludedFiles`? We don't discriminate text vs other resource type in `computeDepsForBundle`, so we can probably just make use of this file for any future Magento-specific ignores we have.

Reply:
Good point, I renamed the file!